tests/phpunit/includes/Private_Access.php

```diff
@@ -38,7 +38,7 @@ trait Private_Access {
 	 * @return mixed Return value of the method call.
 	 * @throws ReflectionException If the object could not be reflected upon.
 	 */
-	private function call_private_method( $object, $method_name, $args = [] ) {
+	protected function call_private_method( $object, $method_name, $args = [] ) {
```

Comment:
Curious, what's the reason for a change here?

Reply:
Now this trait is being used in a Test_Case class, that is a parent class meaning this is not accessible anymore. Private method can not use child classes.